cmd/snapshot_show.go

```diff
@@ -64,14 +78,82 @@ func aptlySnapshotShow(cmd *commander.Command, args []string) error {
 		}
 	}
 
-	withPackages := context.Flags().Lookup("with-packages").Value.Get().(bool)
```

Comment:
I think it's best to leave `withPackages := ...` here instead of 40 lines above, because it's nearer to where it's first used.